src/pages/People.js

```diff
@@ -0,0 +1,407 @@
+import React from "react"
+import Helmet from "react-helmet"
+import s from "styled-components"
+import { Grid, Cell } from "styled-css-grid";
+import "../pages/styles.scss"
+import { Title, Text, Subtitle} from "../components/shared/Typography"
+import { Navbar, Footer } from "../components"
+
+import ArunMajumdar from '../images/arun_majumdar.jpeg'
+import DennisWoodside from '../images/dennis woodside.jpg'
+import ZhengxiaDou from '../images/zhengxia_dou.jpeg'
+import DavdGalligan from '../images/david_galligan.jpeg'
+import JamesHagan from '../images/james_hagan.jpeg'
+import PeterPsarras from '../images/peter_psarras.jpeg'
+import ArthurBenthem from '../images/arthur_benthem.jpeg'
+import ESwann from '../images/e_swann.jpg'
+import AlexBehar from '../images/alex_behar.jpeg'
+import ShuoYang from '../images/shuo_yang.jpeg'
+import AllyWarson from '../images/ally_warson.jpeg'
+import KimberlyZou from '../images/kimberly_zou.jpeg'
+import RyFisher from '../images/ry_fisher.jpeg'
+import FranzHochstrasser from '../images/franz_hochstrasser.jpeg'
+import CaitlynMcCloskey from '../images/caitlyn_mcCloskey.jpeg'
+import BenMay from '../images/ben_may.jpeg'
+import AkaashPadmanabha from '../images/akaash_padmanabha.jpeg'
+import PEGLogo from '../images/peg.jpeg'
+import SSAPLogo from '../images/ssap.png'
+import PUCPLogo from '../images/pucp.jpeg'
+
+const Headshot = s.img`
+    width: 17vw;
+    height: 17vw;
+    object-fit: cover;
+    border-radius: 50%;
+    margin-bottom: 1vw;
+    margin-top: 3vw;
+`
+const Caption = s.p`
+    font-family: Roboto;
+    font-weight: bold;
+    font-size: 20px;
+    text-align: center;
+`
+
+//keynote
+const Keynote = () => {
+  return (
+    <div>
```

Comment:
if this div doesn't have a functional purpose, consider using a React.Fragment (you can write one using <></>)